### PR TITLE
Update 2.how-to-create-subsciption.md

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/nebula-cloud/2.how-to-create-subsciption.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+1. Enter the [Azure Marketplace](https://portal.azure.com/?l=en.en-us#blade/Microsoft_Azure_Marketplace/GalleryMenuBlade/selectedMenuItemId/home), and search for Nebula Graph Cloud in the search bar in Marketplace, or directly click [Nebula Graph Cloud]() to enter the subscription page. [TODO]
+
+2. Select a plan according to your own needs and click `Set up + subscribe`.
+
+   1. In the `Basics` page of **Subscribe Nebula Graph Cloud**, fill in the following plan details:
+
+   - Project details
+
+      |Field|Pre-defined value|
```
Comment (foesa-yang) [inline docs-2.0/nebula-cloud/2.how-to-create-subsciption.md:15]:
```suggestion
      |Field|Description|
```

File: docs-2.0/nebula-cloud/2.how-to-create-subsciption.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      |:---|:---|
+      |Subscription|Select a subscription.|
+      |Resource group|Select an existing resource group or create a new one.|
+
+   - SaaS details
+
+      |Field|Pre-defined value|
```
Comment (foesa-yang) [inline docs-2.0/nebula-cloud/2.how-to-create-subsciption.md:22]:
```suggestion
      |Field|Description|
```

File: docs-2.0/nebula-cloud/2.how-to-create-subsciption.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# How to create a subscription
```
Comment (randomJoe211) [inline docs-2.0/nebula-cloud/2.how-to-create-subsciption.md:1]: We usually use "Create a subscription". Use questions as titles for "What is xxx" topics and FAQs.

**Review by randomJoe211 — DISMISSED** [review]

**Review by randomJoe211 — DISMISSED** [review]

File: docs-2.0/nebula-cloud/2.how-to-create-subsciption.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Before using Nebula Graph Cloud, you need to create a subscription on Azure. This topic describes how to create a subscription on Azure Marketplace.
+
+## Subscription workflow
+
+1. Enter the [Azure Marketplace](https://portal.azure.com/?l=en.en-us#blade/Microsoft_Azure_Marketplace/GalleryMenuBlade/selectedMenuItemId/home), and search for Nebula Graph Cloud in the search bar in Marketplace, or directly click [Nebula Graph Cloud]() to enter the subscription page. [TODO]
+
+2. Select a plan according to your own needs and click `Set up + subscribe`.
+
+   1. In the `Basics` page of **Subscribe Nebula Graph Cloud**, fill in the following plan details:
```
Comment (abby-cyber) [inline docs-2.0/nebula-cloud/2.how-to-create-subsciption.md:11]: If you are talking about something that you are able to view on a page, you will use "ON". So we'd better use On the xxx page, xxx

**Review by abby-cyber — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**